src/main/scala/org/apache/sysml/api/ml/LogisticRegression.scala

```diff
@@ -33,6 +33,12 @@ import org.apache.spark.SparkConf
 import org.apache.spark.mllib.linalg.Vectors
 import org.apache.spark.mllib.regression.LabeledPoint
 import scala.reflect.ClassTag
+import org.apache.spark.sql.types.DataTypes
+import org.apache.spark.sql.RowFactory
+import org.apache.spark.sql.Row
+import scala.collection.mutable.ArrayBuffer
+import org.apache.spark.sql.types.StructField
+import org.apache.sysml.api.ml.util.RDDConverterUtilsExt
```

Comment:
Nitpick: Can you sort these imports to have:
1. `scala.....` at the top, followed by a newline.
2. `org.apache.spark.....` next, followed by a newline.
3. `org.apache.sysml.....` next.